pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TableViewTest.java

```diff
@@ -158,4 +162,57 @@ public void testTableViewUpdatePartitions() throws Exception {
         });
         Assert.assertEquals(tv.keySet(), keys2);
     }
+
+
+    @Test(timeOut = 30 * 1000)
+    // Regression test for making sure partition changes are always periodically checked even after a check returned
+    // exceptionally.
+    public void testTableViewUpdatePartitionsTriggeredDespiteExceptions() throws Exception {
+        String topic = "persistent://public/default/tableview-test-update-partitions";
```

Comment:
We should use another topic name here, this topic name is used in the previous test method.